app/models/measure.rb

```diff
@@ -393,7 +393,10 @@ def verbose_duty_expression
   end
 
   def prettify_generated_duty_expression(duty_expression)
-    duty_expression.sub(/\s%/, '%').sub(/\s\s/, ' ')
+    duty_expression.sub!(/\s\s/, ' ')
+    duty_expression.sub!(/\s%/, '%') if duty_expression.scan(/\d\s%/).present?
+    duty_expression.sub!(/\/\s[a-zA-Z]/, &:downcase) if duty_expression.scan(/\/\s[a-zA-Z]/).present?
```

Comment:
This `if` condition seems redundant? if there's no match then no substitution will occur anyway?